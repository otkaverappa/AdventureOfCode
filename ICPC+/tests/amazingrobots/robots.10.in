12 12
############
#X....#....#
#.....#....#
#...###.#..#
#.......#..#
#...###.#..#
#.....#.#..#
####..#.#.##
#.......#..#
#.....###..#
#.....#.....
############
3
11 11 3 N
11 9 3 E
5 6 3 E
13 13
#############
#............
#.#......#..#
#.#......#..#
#.#......#..#
#.#......#..#
#.#......#..#
#.#......#..#
#.#.....##..#
#.#.........#
#.#.........#
#X#.........#
#############
2
4 12 3 N
2 12 3 W
