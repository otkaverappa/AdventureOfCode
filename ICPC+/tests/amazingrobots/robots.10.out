49
E
E
E
W
S
S
S
S
S
E
E
S
S
E
E
N
N
N
N
N
N
E
N
N
N
N
E
S
S
S
S
S
S
S
S
S
E
E
E
E
E
E
E
E
S
N
E
E
E
